src/universal/modules/teamDashboard/components/Team/Team.js

```diff
@@ -93,6 +102,13 @@ const Team = (props) => {
         <DashHeaderInfo title={DashHeaderInfoTitle}>
           {isSettings ? settingsLinks(teamId) : standardLinks(teamId)}
         </DashHeaderInfo>
+        {isSettings && myTeamMember.isLead &&
```

Comment:
Ah good! I wanted to make certain these controls only showed up when isLead was true.